tests/config/operator_install/olsconfig.crd.openai_introspection.yaml

```
@@ -32,3 +32,10 @@ spec:
       - name: bar_filter
         pattern: '\b(?:bar)\b'
         replaceWith: "openshift"
+  mcp_servers:
+    - name: openshift
+      transport: stdio
+      stdio:
+        command: python
+        args:
+          - /app-root/mcp_local/openshift.py
```

Please use relative path
```suggestion
  mcp_servers:
    - name: openshift
      transport: stdio
      stdio:
        command: python
        args:
          - mcp_local/openshift.py
```